packages/api/src/schema/db-schema.yaml

```diff
@@ -1085,6 +1085,8 @@ components:
           index: true
         outputAssetId:
           index: true
+        projectId:
+          index: true
```

Comment:
don't we need to modify all the task APIs to filter by the context project ID as well?